src/Core/Orders/Factories/OrderFactory.php

```diff
@@ -328,12 +332,18 @@ protected function setFields($order, $fields, $prefix)
      * Sets the user fields.
      *
      * @param Order $order
-     * @return void
+     * @return Order
      */
     protected function setUserFields(&$order)
     {
-        foreach ($this->user->addresses as $address) {
-            $this->setFields($order, $address->fields, $address->billing ? 'billing' : 'shipping');
+        /** @var \Illuminate\Support\Collection $defaultAddresses */
```

Comment:
I guess it's your IDE doing these comments? can we remove them, nothing wrong with them, just we don't use them anywhere else in the codebase AFAIK

Reply:
I'm actually doing them manually, but yes it's for phpstorm - it allows you to use intellisense, etc.

Happy to strip them out if you guys don't have a use for them.

Reply:
Gotcha, maybe I should use phpstorm 🤔